Tools/machines/perlmutter-nersc/Containerfile

```diff
@@ -0,0 +1,151 @@
+# Base System and Essential Tools Installation
```

Comment:
I think traditionally we call this file `Dockerfile` not `Containerfile`, is that still the case?

Reply:
Ah, convention we use because we build with `podman`.